util/conda_package/README.md

```diff
@@ -0,0 +1,61 @@
+# Building Conda package for Open3D
+
+## Prerequisites
+
+Install `conda`, following
+[this instruction](https://conda.io/docs/user-guide/install/index.html).
+
+## Build Conda package
+
+We use the same `setup.py` and shared library objects as when we're building
+the `pip_package`.
+
+### 1. Copy shared libraries
+
+Copy the shared libraries to the corresponding directory inside
+`Open3D/util/pip_package/open3d`. For example, on macOS with Python 3.6,
+`open3d.cpython-36m-darwin.so` shall be copied to
+`Open3D/util/pip_package/open3d/macOS`
+
+### 2. Build Conda package
```

Comment:
This is good for our internal reference.
We need to add documentation for our users whom don't want to build the package from source in order to install it. 

Please take a look here: http://open3d.org/docs/tutorial/Basic/python_interface.html

My recommendation, if you all agree, is to add a section there about installing Open3D for/from Conda. That would mean also uploading a package to Conda, hopefully just one for all OSs.

Reply:
Good suggestion, will add this. I'll first build and upload the package to the Anaconda repository.